README.md

```diff
@@ -39,19 +36,55 @@ If you want to use stackformation globally:
 mv stackformation.phar /usr/local/bin/stackformation
 chmod +x /usr/local/bin/stackformation
 ```
+### Kickstart a demo project
+Imagine we are starting from scratch and we don`t have StackFormation globally available. First of all we have to create a new project directory and move into that new empty directory
+```
+mkdir Demo
+cd Demo
+```
+#### Installation
+Now we have to install StackFormation, for this demo we will install it via composer
+```
+composer require aoepeople/stackformation
+```
+Your first level project structure should be looking like that now
+```
+├── composer.json
+├── composer.lock
+└── vendor
+```
+To check if StackFormation is working properly execute the following command
+```
+vendor/aoepeople/stackformation/src/stackformation.php
```

Comment:
What's composer's default bin dir? `/bin` or maybe `/vendor/bin`? That would look a lot cleaner...